pyatrea/__init__.py

```diff
@@ -2,12 +2,12 @@
 # Author: Juraj Nyiri
 # Tested with:
 # Atrea ECV 280 
-#
 # sw RD5 ver.:
 # 2.01.26
 # 2.01.3O
 # 2.01.32
 #
+# Atrea Duplex 390
```

Comment:
Could you please add your current fw version below the Atrea Duplex 390 similarly to how its above?

Reply:
I'm not sure which version to list here. My control unit (which I guess is RD4) shows these versions:
Driver: 71.16
RD4: 72.06
Digi: 73.17
Web: 74.11

I'm guessing the Web version would be the most relevant here. What do you think?

Reply:
Yes I believe web version would be relevant here, but maybe also others... I think lets list them all, no harm done.

Reply:
Done.